packages/radix-ui-themes/src/helpers/responsive.ts

```diff
@@ -8,7 +8,7 @@ type Responsive<T> = T | Partial<Record<Breakpoints, T>>;
 interface GetResponsiveStylesOptions {
   className: string;
   customProperties: `--${string}`[];
-  value: Responsive<StringOrValue<string>> | Responsive<string> | undefined;
+  value: Responsive<Union<string, string>> | Responsive<string> | undefined;
```

Comment:
These all read weird now, why are they even needed?

Surely `Union<string, string>` is the same as `string` no? So then that'd become `Responsive<string>` which is already in the union next…

Reply:
Technically both `StringOrValue<string>` and `Union<string, string>` is the same as `string`, but TS doesn’t seem to recognize that and complains

I added defaults to the Union type so at least it doesn't read weird, what do you think?
https://github.com/radix-ui/themes/pull/260/commits/42890e6a3dfb065fab8c75c05ed50c5e2d9981c9